docker-compose.yaml

```
@@ -16,19 +16,19 @@ services:
     ports:
     - "80:80"
     depends_on:
-#    - frontend
+    - frontend
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 6 but found 4

(indentation)